docs/appendices/release-notes/unreleased.rst

```diff
@@ -62,6 +62,11 @@ Deprecations
 Changes
 =======
 
+- Added support for CIDR notation in equality comparisons and where clauses,
```

Comment:
Not sure why we imply here that `where clauses` are special? equality comparisons can be used in where clauses. So the former already implies the latter.